sdk/src/main/java/com/hcaptcha/sdk/IHCaptcha.java

```diff
@@ -84,4 +84,8 @@ public interface IHCaptcha {
      */
     HCaptcha verifyWithHCaptcha(@NonNull HCaptchaConfig config);
 
+    /**
+     * Force stop verification and release resources.
+     */
+    void clear();
```

Comment:
Should we call it `reset` instead of `clear` to match the hCaptcha js api `hcaptcha.reset()`?

Reply:
Done